site/landing/content/trademark-usage-policy.md

```diff
@@ -1,13 +1,14 @@
 ---
 title: "OpenTitan Trademark Usage Policy"
+layout: "info"
```

Comment:
Does mdbook support this tags? I thought it was Hugo tags.

Reply:
The main website (`site/landing/`) is still a Hugo site, so we can still use these tags